### ISSUE TITLE
Accessibility issues on release announcement of TAC

### ISSUE THREAD

**Issue by florianduros** [element-hq/element-web#27412]
### Steps to reproduce

- The Announcement is not read out when screenreader is on the TAC button, the `aria-describedby` attribute is missing
- Click on the TAC button doesn't hide the release announcement
